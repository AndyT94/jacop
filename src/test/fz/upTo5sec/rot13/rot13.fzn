array [1..10] of var int: rotated:: output_array([1..10]) = [14,15,16,26,1,2,3,11,12,13];
solve  satisfy;
