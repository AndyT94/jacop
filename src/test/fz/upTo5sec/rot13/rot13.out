rotated = array1d(1..10, [14, 15, 16, 26, 1, 2, 3, 11, 12, 13]);
----------